inc/refresh/refresh.h

```diff
@@ -225,4 +225,8 @@ void    R_BeginFrame(void);
 void    R_EndFrame(void);
 void    R_ModeChanged(int width, int height, int flags);
 
+// Mako - variables for cl_texture macro
+extern vec3_t m_PlayerPos;
+extern vec3_t m_PlayerAngles;
```

Comment:
You can just use cl.refdef.vieworg and cl.refdef.viewangles?
See:
https://github.com/TotallyMehis/q2pro/blob/a66d1b68ae9de47e49e6d1951b92e2ee0d6d0dea/src/client/view.c#L480-L482

Reply:
Probably. I had no idea these existed.
I'll look into it.

Reply:
Fixed in https://github.com/TotallyMehis/q2pro/pull/1/commits/36bc7e9a0f8a8dad96873a98523f9f964e2ad77f